salt/metalk8s/macro.sls

```diff
@@ -3,11 +3,13 @@
 {%- macro pkg_installed(name='') -%}
   {%- set package = repo.packages[name] | default({}) %}
   pkg.installed:
-    - name: {{ name }}
-    - fromrepo: {{ repo.repositories.keys() | join(',') }}
-    {%- if package.version | default(None) %}
+    - name: {{ package.name | default(name) }}
+  {%- if package.version | default(None) %}
```

Comment:
Why this indentation change?

Reply:
IMHO this indentation looks better compare to the previous one (same indentation as the `set` above and only one more indentation than the macro) 

Personnaly I indent jinja action (`{% stuff %}`) independently, I mean indentation not related to the "non jinja" content

Reply:
The indentation is indeed based on jinja and not on yaml.